bigfastapi/notification.py

```diff
@@ -0,0 +1,155 @@
+import fastapi
+from fastapi import Request, APIRouter
+from datetime import datetime
+import sqlalchemy.orm as orm
+from bigfastapi import auth
+from bigfastapi.db.database import get_db
+from fastapi import Depends
+
+from bigfastapi.schemas import users_schemas
+from .models import notification_models
```

Comment:
import this as model

Reply:
So I should import notification_models from models as model right?